cmd/pipeline/main.go

```diff
@@ -277,9 +278,14 @@ func main() {
 	})
 
 	// insert shared organization to DB if not exists
-	sharedOrg := auth.Organization{Name: viper.GetString(config.SpotguideSharedLibraryGitHubOrganization)}
-	if err := db.Where(sharedOrg).FirstOrCreate(&sharedOrg).Error; err != nil {
+	sharedOrgName := viper.GetString(config.SpotguideSharedLibraryGitHubOrganization)
+	if org, _, err := github.NewClient(nil).Organizations.Get(context.Background(), sharedOrgName); err != nil {
 		log.Errorf("failed to create shared organization: %s", err.Error())
```

Comment:
If the shared spotguide insertion into db is required by Pipeline than we should exit with error if the insertion fails.